planning/behavior_path_planner/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ autoware_package()
 find_package(OpenCV REQUIRED)
 find_package(magic_enum CONFIG REQUIRED)
 
-set(COMPILE_WITH_OLD_ARCHITECTURE TRUE)
+set(COMPILE_WITH_OLD_ARCHITECTURE FALSE)
```

Comment:
This flag shouldn't set `FALSE` yet.

Reply:
sorry, I purposely set the flag to flase due to wanting to test the lane change behavior on evaluator. 
Since all scenario passes, this commit https://github.com/autowarefoundation/autoware.universe/pull/3016/commits/f02aa4c42595bc86d8e9432cdde27ff6c53e8da7 set the value to TRUE

Reply:
:ok_hand: